lib/express-logger.js

```diff
@@ -82,11 +82,14 @@ module.exports = function (options) {
     return audit;
 };
 
-//convert all options fields that need to be array by default
+// Convert all options fields that need to be array by default
```

Comment:
i think that we can completely drop this code and mention it in the changelog as part of the breaking changes. i don't see any reason for maintaining this code.

Reply:
thought about it a bit more.

don't want to remove the entire logic, since unexpected behavior can still happen when passing non array value to an array field.

my solution is to still have the code, but not default to array in such cases, just leave a default empty array with a warning. 
I think ignoring it completely is wrong since it may have implications on masking of fields if not configured correctly

Reply:
wdyt?